lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -280,13 +281,24 @@ def add_job_logs(self, job_id, log_lines):
             ts = input_line.get("ts")
             # TODO Maybe use strpos for efficiency?
             if ts is not None:
-                try:     # input ts as float/int/str epoch timestamp
-                    ts = float(ts)
-                except Exception:
-                    try:  # input ts as datetime string
-                        ts = dateutil.parser.parse(ts).timestamp()
+                if isinstance(ts, int):  # input ts as epoch in milliseconds
```

Comment:
It's most likely going to be a string, so maybe put that as the first if?
Should we have test cases for each of these instances?

Reply:
👍 sounds good. Added different input time formats in the test.